fast64_internal/oot/oot_constants.py

```diff
@@ -1,1553 +1,1565 @@
-ootEnumMeshType = [
-	#("Custom", "Custom", "Custom"),
-	("0", "Type 0 (Simple)", "Type 0 (Simple)"),
-	("1", "Type 1 (Prerendered)", "Type 1 (Prerendered)"),
-	("2", "Type 2 (Distance Culling)", "Type 2 (Distance Culling)"),
+ootEnumRoomShapeType = [
+    # ("Custom", "Custom", "Custom"),
+    ("ROOM_SHAPE_TYPE_NORMAL", "Normal", "Normal"),
+    ("ROOM_SHAPE_TYPE_IMAGE", "Image", "Image"),
+    ("ROOM_SHAPE_TYPE_CULLABLE", "Cullable", "Cullable"),
+]
+
+ootRoomShapeStructs = [
+    "RoomShapeNormal",
+    "RoomShapeCullable",
+]
+
+ootRoomShapeEntryStructs = [
+    "RoomShapeDListsEntry",
+    "RoomShapeCullableEntry",
 ]
```

Comment:
I think these as dicts would be easier to understand/maintain than the 0/1 indexing
e.g.
```py
ootRoomShapeStructs = {
    "ROOM_SHAPE_TYPE_NORMAL": "RoomShapeNormal",
    "ROOM_SHAPE_TYPE_CULLABLE": "RoomShapeCullable",
}
```
and
```py
ootRoomShapeStructs[mesh.roomShape]
```